### PR TITLE
Add netlify.yml file

### PR DESCRIPTION
**Pull request by angelocordon**
Add a build file to configure Netlify deploys outside of the UI. 

More importantly, this adds a temporary `SKIP_PREFLIGHT_CHECK` env variable for Netlify to skip dependency checks between react script's dependency on Jest 24 vs what we have (or will have: a9873afecff38512d6b8e4d98d9d04f32bf3de32).

This should allow for our Github action tests to pass with Netlify deploys.

### CODE REVIEW

**Comment by angelocordon** [comment]
**tl;dr:** CRA's react-script's requires a dependency on Jest 24, and we need Jest 25 in order for React Testing Library to work correctly. The two different dependency versions between what we have and what react-script requires causes Netlify to send a failing test back to Github. I think it still deploys as expected, but it's reporting our PRs to fail.

In order to mitigate this, I've added an environment variable that's only really used within Netlify. From the error logs, it was saying to use a `.env` file but it's not something I want us to permanently have. It seems like a good fix for now, but also it's making me think to prioritize Next exploration sooner rather later.

**Comment by lpatmo** [comment]
> and we need Jest 25 in order for React Testing Library to work correctly

Note that we found we needed to upgrade to Jest 25 in https://github.com/codebuddies/frontend/pull/102

**Comment by lpatmo** [comment]
Belated question for @angelocordon: are you adding the same `SKIP_PREFLIGHT_CHECK=true` in a local `.env` file?

image

**Comment by lpatmo** [comment]
(Got that after I pulled in the latest from master and rebased)

**Comment by angelocordon** [comment]
Is that from running npm install?

**Comment by lpatmo** [comment]
This happened after I ran `npm install` and `npm run start` without a `.env` file with `SKIP_PREFLIGHT_CHECK=true` inside it.

**Comment by lpatmo** [comment]
Are you getting the same thing?
